pytorch_blade/src/torch-mlir/lib/Conversion/TorchToMhlo/TorchToMhlo.cpp

```diff
@@ -560,6 +556,19 @@ class ConvertAtenDivOp : public OpConversionPattern<AtenOpT> {
     auto rhsTensor = rhsTy ? rhs : rhsAsTensor;
     rhsTy = rhsTensor.getType().dyn_cast<TensorType>();
 
+    auto rhsElemTy = rhsTy.getElementType();
+    if (lhsElemTy != rhsElemTy) {
+      auto lhsScalarType = getScalarTypeForType(lhsElemTy);
+      auto rhsScalarType = getScalarTypeForType(rhsElemTy);
+      auto promotedType =
+          torch_upstream::promoteTypes(lhsScalarType, rhsScalarType);
+      auto elemType = getTypeForScalarType(op.getContext(), promotedType);
+      lhs = rewriter.create<mhlo::ConvertOp>(op.getLoc(), lhs, elemType);
```

Comment:
it is better to check if `promotedType` equals one of `lhs`'s or `lhs`'s to avoid one converting?